### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mabicca**
Adding note about installing from source not being supported.

## Description

Issue # 

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,9 @@ You can view more installation options by running:
 
 The agent's log file is kept at `/var/log/waagent.log`.
 
+## NOTE
+Installing the waagent from source **is not supported**
+
```
Comment (narrieta) [inline README.md:111]: I would propose something similar to "Installing from source is meant to be used in advanced scenarios, such as creating a new VM image, and support is very limited (open an issue here and we may be able to provide guidance). Do not install from source while troubleshooting issues with the Agent."

Comment (mabicca) [reply README.md:111]: What about we revamp that section to specifically talk about installing from source when creating a new, customized image and explaining that installing from source is not supported on Azure Endorsed Distributions? At least that would be clear in terms of the support in those scenarios where we should leverage the distribution package management.

Comment (narrieta) [reply README.md:111]: ok, can you post how it would read?

Comment (mabicca) [reply README.md:111]: Sure, could we add a note saying:

Installing the waagent from the source is to be used only for new virtual image creation (custom images) and the [support](https://docs.microsoft.com/en-us/troubleshoot/azure/cloud-services/support-linux-open-source-technology) is very limited for those scenarios. 

For [Endorsed Linux distributions on Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros) images, the only supported method of installation is using the distribution's own packaging mechanisms such as apt, yum or zypper.

Comment (mabicca) [reply README.md:111]: @narrieta let me know if this is good to update.

Comment (mabicca) [reply README.md:111]: @narrieta I know it's been a while, but not sure if we were able to update the README about this?

Comment (narrieta) [reply README.md:111]: @mabicca sorry for the delay. Submitted #2941

**Comment by narrieta** [comment]
Replaced by #2941

**Pull request closed without merging**